src/modules/launcher/Plugins/Microsoft.Plugin.Folder.UnitTests/InternalQueryFolderTests.cs

```diff
@@ -48,6 +48,7 @@ public void Query_ThrowsException_WhenCalledNull()
         [DataRow(@"c", 0, 0, false, DisplayName = "String empty is nothing")]
         [DataRow(@"c:", 2, 1, false, DisplayName = "Root without \\")]
         [DataRow(@"c:\", 2, 1, false, DisplayName = "Normal root")]
+        [DataRow(@"c:/", 2, 1, false, DisplayName = "Normal root with /")]
```

Comment:
We should test all cases: `/users`, `C:/`, `C:/Users/Public`

Reply:
I have changed the test to cover the full case. Note that this is just testing the Query.

Queries starting with `/`, `\`, `~` are expanded by `FolderHelper.Expand` but could be hard to test in CI since it's relying on the system environment variables: `Path.GetPathRoot(Environment.SystemDirectory)` and `Environment.GetFolderPath(Environment.SpecialFolder.UserProfile)`.